css/main.css

```diff
@@ -42,6 +42,20 @@ body {
 	background-color: white;
 }
 
+.nav-active {
```

Comment:
This styles a class of nav-active that does not exist in the nva-bar.php file. So this does not appear on the site.